source/install/prod-ubuntu-16.04.rst

```diff
@@ -0,0 +1,489 @@
+        mattermost=> \q
+
+
+Set up Mattermost Server
+------------------------
+
+1. For the purposes of this guide we will assume this server has an IP
+   address of ``10.10.10.2``
+2. For the sake of making this guide simple we located the files at
+   ``/home/ubuntu/mattermost``.
+3. We have also elected to run the Mattermost Server as the ``ubuntu``
+   account for simplicity. We recommend setting up and running the
+   service under a ``mattermost`` user account with limited permissions.
+4. Download `any version of the Mattermost Server <https://docs.mattermost.com/administration/upgrade.html#version-archive>`_ by typing::
+
+       wget https://releases.mattermost.com/X.X.X/mattermost-X.X.X-linux-amd64.tar.gz
+
+   Where ``X.X.X`` is typically the latest Mattermost release version.
+
+5. Unzip the Mattermost Server by typing::
+
+       tar -xvzf mattermost-X.X.X-linux-amd64.tar.gz
+
+6. Create the storage directory for files. We assume you will have
+   attached a large drive for storage of images and files. For this
+   setup we will assume the directory is located at
+   ``/mattermost/data``.
+
+   -  Create the directory by typing::
+
+       sudo mkdir -p /mattermost/data
+
+   -  Set the ubuntu account as the directory owner by typing::
+
+       sudo chown -R ubuntu /mattermost
+
+7. Configure Mattermost Server by editing the config.json file at
+   ``/home/ubuntu/mattermost/config``:
+
+   -  Change to the directory::
+
+       cd ~/mattermost/config
+
+   -  Edit the file by typing::
+
+       vi config.json
+
+   -  replace ``DriverName": "mysql"`` with ``DriverName": "postgres"``
+   -  replace
+      ``"DataSource": "mmuser:mostest@tcp(dockerhost:3306)/mattermost_test?charset=utf8mb4,utf8"``
+      with
+      ``"DataSource": "postgres://mmuser:mmuser_password@10.10.10.1:5432/mattermost?sslmode=disable&connect_timeout=10"``
+   -  Optionally you may continue to edit configuration settings in
+      ``config.json`` or use the System Console described in a later
+      section to finish the configuration.
+
+8. Test the Mattermost Server
+
+   -  Change to the folder with the executable::
+
+       cd ~/mattermost/bin
+
+   -  Run the Mattermost Server by typing::
+
+       ./platform
+
+   -  You should see a console log like ``Server is listening on :8065``
+      letting you know the service is running.
+   -  Stop the server for now by typing ``ctrl-c``
+
+9. Create a systemd unit-file for service management.
+
+   - Create the unit-file::
+
+       sudo vi /lib/systemd/system/mattermost.service
+
+   - Fill it with the following content::
+
+         [Unit]
+         Description=Mattermost
+         After=network.target
+         After=postgresql.service
+         Requires=postgresql.service
+
+         [Service]
+         Type=simple
+         ExecStart=/home/mattermost/mattermost/bin/platform
+         Restart=always
+         RestartSec=10
+         WorkingDirectory=/home/mattermost/mattermost
+         User=mattermost
```

Comment:
If we are using user and group `mattermost` they should be created at some point as I commented above